modelforge/tests/test_dataset.py

```diff
@@ -484,89 +483,84 @@ def test_dataset_neighborlist(model_name, single_batch_with_batchsize_64):
     # pairlist is in ascending order in row 0
     assert torch.all(pair_list[0, 1:] >= pair_list[0, :-1])
 
-    if model_name == "ANI1x":
-        # test the pairlist for the ANI potential (whith non-redundant atom pairs)
-        # TODO
-        pass
-    else:
-        # test the pairlist for message passing networks (with redundant atom pairs)
-        # first molecule is methane, check if bonds are correct
-        methan_bonds = pair_list[:, :20]
+    # test the pairlist for message passing networks (with redundant atom pairs)
+    # first molecule is methane, check if bonds are correct
+    methan_bonds = pair_list[:, :20]
```

Comment:
just a minor typo, might want this to be `methane_bonds`

Reply:
Thanks! Corrected the typo.